### DISCUSSION TITLE
Was bedeutet Fehlercode: 47 "FB Überstrom"?

### DISCUSSION THREAD

**Discussion by themanfrommoon** [tbnobody/OpenDTU#1536] [Q&A]
Moin,

Was bedeutet Fehlercode: 47 "FB Überstrom"?

.....
facebook Überstrom?
Fernbedienung Überstrom?
Fachbereich Übertstrom?
Fussball Überstrom?
Flugbetrieb Überstrom?
Frequenz B....b.....bbb........Bereich?! nee, passt nicht...ich hab keine Ahnung.

Ich habe keine Idee was wirklich gemeint ist.

....ich mag Abkürzungen nicht.......weil jeder denkt, was anderes ist gemeint und keiner weiss was wirklich los ist.

Bitte Abkürzungen vermeiden, weil die gerne mal unbemerkt in die Irre führen.

Danke für die Aufklärung und beste Grüße,
Chris

**Comment by stefan123t** [comment]
Vermutlich steht FB für den Flyback Transformer im Hoymiles Wechselrichter. Das sind die Übersetzungen bzw Abkürzungen des Herstellers. Bitte dort beschweren oder nachfragen wofür es wirklich steht.
Siehe https://github.com/tbnobody/OpenDTU/issues/121#issuecomment-1753791623

**Reply by stefan123t** [reply]
Die Abkürzung `FB` wird hier https://github.com/tbnobody/OpenDTU/discussions/590#discussioncomment-7224435 in ingesamt vier Events vom Hersteller verwendet. Die komplette Liste der Events aus der `warn_code.json` der *Hoymiles S-Miles Installer App* findet man hier https://github.com/tbnobody/OpenDTU/issues/121#issuecomment-1752124208

Vielleicht ließe sich auch noch etwas aus den chinesischen Originalbezeichnungen der Events ableiten, wofür das FB tatsächlich steht ?
Das hatte @tbnobody seinerzeit https://github.com/tbnobody/OpenDTU/discussions/590#discussioncomment-7251523 für andere Events erfolgreich ausgeknobelt.

Event ID | 	Description
--- | ---
46 | 	FB overvoltage
47 | 	FB overcurrent
48 | 	FB clamp overvoltage
49 | 	FB clamp overvoltage

@themanfrommoon kannst Du mal eine chinesische Anleitung der Events und Handlungsanweisung für Deinen Wechselrichter von Hoymiles aus dem Chinesischen übersetzen (z.B. mit Google Translate) und das nachprüfen ?
Häufig ist es dabei sinnvoll sich die Zeichen auch einmal einzeln, bzw. in kleinen Gruppen übersetzen zu lassen, da man dann einen engeren Bedeutungskontext der Zeichen hat und nicht nur den üblichen Zusammenhang.

Aber eventuell steht sogar in der original chinesischen Event-Beschreibung `FB` drin, ich meine das nämlich nachgeprüft zu haben und konnte mir auch keinen Reim darauf machen, außer eben der von @homeautomation genannte Flyback-Transformer der im Hoymiles HM m.W. zum Current-Sensing (?) verbaut ist. Da müsste dann die entsprechende Leitung durchgehen, damit der induktiv den Strom messen kann.

**Comment by themanfrommoon** [comment]
Ich finde leider keine chinesische Anleitung?!
